### PR TITLE
Adding PyGameZero Workshop

### PR DESCRIPTION
**Pull request by Cheukting**

### CODE REVIEW

File: src/data/links.json [hunk 1]
```diff
@@ -60,7 +60,8 @@
           "name": "C API Summit",
           "path": "/capi"
         },
-        { "name": "Humble Data", "path": "/humble-data" }
+        { "name": "Humble Data", "path": "/humble-data" },
+        { "name": "PyGameZero Workshop", "path": "/pygamezero" }
```
Comment (cybit) [inline src/data/links.json:64]: The links order changed. Could you please update it to follow this order?
```
- Conference events by day, i.e. events that happen once, during the
conference, such as workshop, summit, etc.
- Social events by day, i.e. events that happen once outside of the main
conference.
- Multiple events / events that span several days, such as the PyLdadies
programme, hackathon, etc.
This is just a suggestion and there could be many more sensible way to
order this.
```

File: src/content/pages/pygamezero.mdx [hunk 2]
```diff
@@ -0,0 +1,36 @@
+---
+title: PyGameZero Workshop
+subtitle: An introduction and fun workshop to teach beginners how to using Python to make games.
```
Comment (cybit) [inline src/content/pages/pygamezero.mdx:3]:
```suggestion
subtitle: An introductionary and fun workshop to teach beginners how to use Python to create games.
```

File: src/content/pages/pygamezero.mdx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+# PyGameZero Workshop
+Are you a child at heart? Do you want to make your own game using Python? This workshop is for you! In this short introduction workshop, we will learn about PyGameZero, a simple framework in Python that allow any coding beginners to create a simple game using Python.
```
Comment (cybit) [inline src/content/pages/pygamezero.mdx:7]:
```suggestion
Are you a child at heart? Do you want to make your own game using Python? This workshop is for you! In this short workshop, we will learn about PyGameZero - a simple framework in Python that allow coding beginners to create a simple game using Python.
```

File: src/content/pages/pygamezero.mdx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+**When**: Wednesday 10th July 2024 10:45
+**Where**: Open Space at EuroPython
+**Who**: EuroPython attendees (no extra registration required)
```
Comment (cybit) [inline src/content/pages/pygamezero.mdx:11]:
```suggestion
**Who**: Anyone with a valid in-person EuroPython 2024 ticket - Conference or Combined (see [ticket types (https://ep2024.europython.eu/tickets#ticket-types) for details) - No extra registration required!
```

File: src/content/pages/pygamezero.mdx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+## FAQ
+
+- *I am a beginner in coding/ Python, can I join?*
```
Comment (cybit) [inline src/content/pages/pygamezero.mdx:15]:
```suggestion
- *I am a beginner in coding / Python, can I join?*
```

File: src/content/pages/pygamezero.mdx [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+Sure! This workshop is for beginners in coding and Python, just bring your computer!
```
Comment (cybit) [inline src/content/pages/pygamezero.mdx:17]:
```suggestion
Sure! This workshop is for beginners in coding and Python. Just bring your computer!
```

File: src/content/pages/pygamezero.mdx [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+- *I have kid(s) with me at the conference, can I bring my kid(s)?*
```
Comment (cybit) [inline src/content/pages/pygamezero.mdx:19]:
```suggestion
- *I have kid(s) with me at the conference. Can I bring my kid(s)?*
```

File: src/content/pages/pygamezero.mdx [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+If your kid(s) is 7-12 years old and you will be accompanying them, yes, you are welcome to bring your kid(s).
```
Comment (cybit) [inline src/content/pages/pygamezero.mdx:21]:
```suggestion
If your kid(s) is/are 7-12 years old  - and you will be accompanying them -, then yes, you are welcome to bring your kid(s).
```

File: src/content/pages/pygamezero.mdx [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+- *What do I need to bring?*
+A charged computer that can connect to the internet with wifi, the creative energy and an open mind to learn.
```
Comment (cybit) [inline src/content/pages/pygamezero.mdx:24]:
```suggestion
A charged computer that can connect to the internet with wifi, the creative energy, and an open mind to learn.
```

**Review by cybit — DISMISSED** [review]

File: src/data/links.json (hunk 1 above)
Comment (cybit) [reply src/data/links.json:64]: This is the blocker. The rest of the comments are non-blocking.

File: src/content/pages/pygamezero.mdx (hunk 3 above)
Comment (laisbsc) [reply src/content/pages/pygamezero.mdx:7]: Maybe add the duration of the workshop?

> In this **X hour-long** workshop, (...)

Or something on these lines :)

**Review by laisbsc — CHANGES_REQUESTED** [review]
Looks really good! TY for putting it together. Is this going to happen at an Open Space?

**Comment by cybit** [comment]
@Cheukting any updates on your side here? Could you please to the rebase?

**Comment by Cheukting** [comment]
PR updated, please final check

File: src/data/links.json [hunk 10]
```diff
@@ -60,6 +60,7 @@
           "name": "Speakers' Dinner",
           "path": "/speakers-dinner"
         },
+        { "name": "PyGameZero Workshop (Wednesday)", "path": "/pygamezero" },
```
Comment (egeakman) [inline src/data/links.json:63]:
```suggestion
        {
           "name": "PyGameZero Workshop (Wednesday)",
           "path": "/pygamezero",
         },
```

**Review by egeakman — APPROVED** [review]

File: src/data/links.json [hunk 11]
```diff
@@ -60,6 +60,10 @@
           "name": "Speakers' Dinner",
           "path": "/speakers-dinner"
         },
+        {
+          "name": "PyGameZero Workshop (Wednesday)",
+          "path": "/pygamezero"
```
Comment (cybit) [inline src/data/links.json:65]: Again: Please move this up. This is not a social event.  I asked for this to be moved accordingly before.

```
- Conference events by day, i.e. events that happen once, during the
conference, such as workshop, summit, etc.
- Social events by day, i.e. events that happen once outside of the main
conference.
- Multiple events / events that span several days, such as the PyLdadies
programme, hackathon, etc.
```

File: src/content/pages/pygamezero.mdx [hunk 12]
```diff
@@ -0,0 +1,36 @@
+---
+title: PyGameZero Workshop
+subtitle: An introductionary and fun workshop to teach beginners how to use Python to create games.
+---
+
+# PyGameZero Workshop
+Are you a child at heart? Do you want to make your own game using Python? This workshop is for you! In this 2-hour long workshop, we will learn about PyGameZero - a simple framework in Python that allow coding beginners to create a simple game using Python.
```
Comment (cybit) [inline src/content/pages/pygamezero.mdx:7]:
```suggestion
Are you a child at heart? Do you want to make your own game using Python? This workshop is for you! In this 2-hour long workshop, we will learn about PyGameZero - a simple framework in Python that allows coding beginners to create a simple game using Python.
```

File: src/content/pages/pygamezero.mdx [hunk 13]
```diff
@@ -0,0 +1,36 @@
...
+
+Sure! This workshop is for beginners in coding and Python. Just bring your computer!
+
+- *I have kid(s) with me at the conference. Can I bring my kid(s)?*
+
+If your kid(s) is/are 7-12 years old  - and you will be accompanying them -, then yes, you are welcome to bring your kid(s).
+
+- *What do I need to bring?*
+
+A charged computer that can connect to the internet with wifi, the creative energy, and an open mind to learn.
```
Comment (cybit) [inline src/content/pages/pygamezero.mdx:26]:
```suggestion
A charged computer that can connect to the internet with wifi, creative energy, and an open mind to learn.
```

**Review by cybit — CHANGES_REQUESTED** [review]

**Review by cybit — APPROVED** [review]

**Pull request merged**